### PR TITLE
fix: update pre-render to work with new razzle export

### PR DESCRIPTION
**Pull request by fivethreeo**
Can you fix this line? :)

https://github.com/jaredpalmer/after.js/compare/feature/pre-render...fivethreeo:feature/pre-render?expand=1#diff-d60a939e72f08f2ab2ab53676f361cd9R34

### CODE REVIEW

**Comment by fivethreeo** [comment]
Got some issues with the pre-render feature branch, before and after my changes. Master works fine. Even tried examples/basic. Same issue, "Invariant failed: You sh…ink> outside a <Router>"

**Comment by fivethreeo** [comment]
Was yarn link issue ofcourse. :)

**Comment by fivethreeo** [comment]
Switched to yalc, now is slow in the browser for some reason.

**Comment by fivethreeo** [comment]
No data is returned here:

https://github.com/jaredpalmer/after.js/pull/376/files#diff-074d31de12ee8ff11f724a2e48b695deR17

Can't figure out why.

File: examples/with-prerender/razzle.config.js [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 module.exports = {
   experimental: {
-    prerender: true,
+    static_export: {
+      window_routes_variable: 'AFTER_STATIC_ROUTES',
```
Comment (jaredpalmer) [inline examples/with-prerender/razzle.config.js:6]: These should just be defaults.

File: examples/with-prerender/src/static_export.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { renderStatic } from '@jaredpalmer/after';
+import afterRoutes from './routes';
+
+const assets = require(process.env.RAZZLE_ASSETS_MANIFEST);
+const chunks = require(process.env.RAZZLE_CHUNKS_MANIFEST);
+
+export const render = async (req, res) => {
+  try {
+    const { html, data } = await renderStatic({
```
Comment (jaredpalmer) [inline examples/with-prerender/src/static_export.js:9]: Is there no way for us keep the original render method for static? Our goal should be minimal amount of changes required

File: examples/with-prerender/razzle.config.js (hunk 1 above)
Comment (jaredpalmer) [reply examples/with-prerender/razzle.config.js:6]: Oh I see. These are razzle configs.

File: examples/with-prerender/src/static_export.js (hunk 2 above)
Comment (fivethreeo) [reply examples/with-prerender/src/static_export.js:9]: Maybe do something like a ssgContext argument that after sets html and data on?

Comment (fivethreeo) [reply examples/with-prerender/src/static_export.js]: Fixed all the issues now :)

File: packages/after.js/src/After.tsx [hunk 3]
```diff
@@ -114,7 +114,15 @@ class Afterparty extends React.Component<AfterpartyProps, AfterpartyState> {
...
       // in ssg mode we don't call component.getInitialProps
       // instead we fetch the page-data.json file
-      const loadData = ssg ? loadStaticProps : loadInitialProps;
+      const useStaticProps = ssg
+        ? (window as any).AFTER_STATIC_ROUTES
+          ? (window as any).AFTER_STATIC_ROUTES.includes(
+              location.pathname.replace(/^\/|\/$/g, '')
+            )
+          : true
+        : false;
```
Comment (nimaa77) [inline packages/after.js/src/After.tsx:123]: what is the use case of this? (I'm just asking)

**Pull request closed without merging**